src/array.cr

```diff
@@ -1507,6 +1507,18 @@ class Array(T)
     ary
   end
 
+  # Removes the first value of `self`, at index 0. This method returns the removed value.
+  # Returns `nil` without raising any error if array is of 0 size.
```

Comment:
I'd turn this sentence around, "If the array is empty..."

Reply:
It would make sense, I just copied it from the documentation of the question-mark-less method. Should I change it there too?

Reply:
Yes please.

Reply:
Thanks for the feedback, I just pushed the change.